the-collab-lab/smart-shopping-list-deprecated
# Set up collab lab package

## Description

This removes two files from the `src/lib` directory and instead sources the functions in those files from a [new NPM package](https://www.npmjs.com/package/@the-collab-lab/shopping-list-utils) for The Collab Lab. Now, it will be easy to update these methods and add other functionality without having to update other repositories.

## Acceptance Criteria

Methods are still identical to the code in the repository so there should be no changes. Methods can be called from an import now:

```
import { getToken, words, calculateEstimate } from '@the-collab-lab/shopping-list-utils
```

## Type of Changes

A little bit of refactoring, added tests for the `calculateEstimate` function, and updated dependencies!

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    ✓| :hammer: Refactoring       |
|   ✓ | :100: Add tests            |
|    ✓| :link: Update dependencies |
|    | :scroll: Docs              |

## Testing Steps / QA Criteria

You can check out the [new repository on Github](https://github.com/the-collab-lab/shopping-list-utils), with a readme that includes details on how to run and test the package.
